docs/project-list.md

```diff
@@ -0,0 +1,103 @@
+---
+date-generated: 2020-03-24T14:44:10
+tool: NuGetTasks.GenerateMarkdownDoc
+---
+
+
+# NuGet Project Overview
+
+Below is a list of all source code projects for NuGet libraries and supported NuGet clients
+
```

Comment:
Lots of unnecessary empty space here :)

Reply:
Removed some empty lines. Added double empty lines between sections. Do you want exactly one empty line between sections ?